### ISSUE TITLE
Do we need a special API key from a non-free account for using Chat GPT4 in Bavarder? 

### ISSUE THREAD

**Issue by jbingram** [Bavarder/Bavarder#16]
**Describe the bug**
I cannot use ChatGPT 4 in Bavarder with my basic API key from my free openAI account.
Is it the expected behavior for Bavarder?
Do we need to use here a special API key by paying a $20/month ChatGPT-plus account?
If this is the case, then this should be stated for instance in the section Preferences/Providers/OpenAI GPT 4 (below entering the API key). 

** Suggestion of modification**
You could add for instance a question mark symbol on the left side of the edit Api key icon (the pen)

**Comment by 0xMRTT** [comment]
Yes, it requires the paying plan. But if you don't pay and use a free plan (like me). You will see 

![image](https://user-images.githubusercontent.com/105598867/236406007-4d201c78-62e3-4fd9-9676-e8789af77ac0.png)

But anyway, i added an explaination 

![image](https://user-images.githubusercontent.com/105598867/236406156-6f38b1e8-3482-454b-8695-f574bf40ac06.png)

**Comment by jbingram** [comment]
thx, much appreciated!
